vip-mail.php

```diff
@@ -102,7 +102,11 @@ public function phpmailer_init( &$phpmailer ): void {
 			return;
 		}
 
-		if ( 'smtp' === $phpmailer->Mailer ) {
+		$host_overwrite_allow_list = defined( 'VIP_SMTP_HOST_OVERWRITE_ALLOW_LIST' )
+			? explode( ',', constant( 'VIP_SMTP_HOST_OVERWRITE_ALLOW_LIST' ) )
+			: [];
+
+		if ( 'smtp' === $phpmailer->Mailer && ! in_array( $phpmailer->Host, $host_overwrite_allow_list ) ) {
```

Comment:
```suggestion
		if ( 'smtp' === $phpmailer->Mailer && ! in_array( $phpmailer->Host, $host_overwrite_allow_list, true ) ) {
```

In this context, there are very few scenarios where the use of strict comparison with `in_array` (`true` as the third parameter) would cause issues. So it could make sense to set it up to avoid any doubts this code does something you don't expect.